aeon_crud.proto

```diff
@@ -1,32 +1,17 @@
 syntax = "proto3";
 
-package aeon_router;
+import "aeon_error.proto";
+import "aeon_value.proto";
+import "aeon_schema.proto";
 
-option go_package = "./pb";
+package aeon;
 
-// API to Aeon - a distributed database based on Tarantool.
-service AeonRouterService {
-  //
-  // Diagnostic requests.
-  //
-
-  // Pings the router.
-  rpc Ping(PingRequest) returns (PingResponse) {}
-
-  //
-  // DDL requests.
-  //
+option go_package = "github.com/tarantool/aeon/aeon/grpc/server/pb";
```

Comment:
Decided to remove this for now and instead provide a helper script that would generate pb.go files taking the package name as an argument.